pkg/server/sandbox_run_test.go

```diff
@@ -474,21 +474,14 @@ func TestTypeurlMarshalUnmarshalSandboxMeta(t *testing.T) {
 	}
 }
 
-func TestHostPrivilegedSandbox(t *testing.T) {
+func TestHostAccessingSandbox(t *testing.T) {
```

Comment:
Let's keep both privleged and nonPrivleged, and both of them should return true. This proves that the change takes effect.